static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -130,6 +149,162 @@ function NoContext({isLoading}: NoContextProps) {
   );
 }
 
+type ReleaseContextProps = {
+  api: Client;
+  dataRow: EventData;
+  organization: Organization;
+};
+
+function ReleaseContext(props: ReleaseContextProps) {
+  const {isLoading, isError, data} = useQuery<ReleaseWithHealth>(
+    [`/organizations/${props.organization.slug}/releases/${props.dataRow.release}/`],
+    undefined,
+    {
+      staleTime: fiveMinutesInMs,
+      retry: false,
+    }
+  );
+
+  const getCommitAuthorTitle = () => {
+    const user = ConfigStore.get('user');
+    const commitCount = data?.commitCount || 0;
+    let authorsCount = data?.authors.length || 0;
+
+    const userInAuthors =
+      data &&
+      data.authors.length >= 1 &&
+      data.authors.find((author: User) => author.id && user.id && author.id === user.id);
+
+    if (userInAuthors) {
+      authorsCount = authorsCount - 1;
+      return data.authors.length - 1 !== 1 && data.commitCount !== 1
+        ? tct('[commitCount] commits by you and [authorsCount] others', {
+            commitCount,
+            authorsCount,
+          })
+        : data.commitCount !== 1
+        ? tct('[commitCount] commits by you and [authorsCount] other', {
+            commitCount,
+            authorsCount,
+          })
+        : data.authors.length - 1 !== 1
+        ? tct('[commitCount] commit by you and [authorsCount] others', {
+            commitCount,
+            authorsCount,
+          })
+        : tct('[commitCount] commit by you and [authorsCount] other', {
+            commitCount,
+            authorsCount,
+          });
```

Comment:
```suggestion
      return authorsCount !== 1 && data.commitCount !== 1
        ? tct('[commitCount] commits by you and [authorsCount] others', {
            commitCount,
            authorsCount,
          })
        : data.commitCount !== 1
        ? tct('[commitCount] commits by you and 1 other', {
            commitCount,
          })
        : authorsCount !== 1
        ? tct('1 commit by you and [authorsCount] others', {
            authorsCount,
          })
        : t('1 commit by you and 1 other');
```

I think we can simplify this in the later cases after some of the checks